src/addin/client-interfaces/addin-action-button-config.ts

```diff
@@ -0,0 +1,16 @@
+export interface AddinActionButtonConfig {
+  /**
+   * The title of the action button.
+   */
+  title?: string;
```

Comment:
`AddinClientReadyArgs` already has a `title` property.  Can we use that one instead?

Reply:
Yep - just forgot to remove